hack/lib/common-var-init.sh

```diff
@@ -236,6 +236,10 @@ nameserver ${UPSTREAM_DNS_IP}
 EOF
   fi
 fi
+#
+# For starting mizar-operator pods on scale-out TP servers successfully on Ubuntu 20.04
+#
+MIZAR_OPERATOR_HOST_PATH=${MIZAR_OPERATOR_HOST_PATH:-"/etc/kubernetes"}
```

Comment:
This is only used in arktos-up-scale-out-poc.sh, let's just create the path there. Not suitable for common variables.

Reply:
yes.